laravel/resources/js/admin.js

```diff
@@ -56,37 +56,6 @@ $(document).ready(function()
         $(this).parents('.volunteer').find('.buttons').style({visibility: 'visible', opacity: 1});
     });
 
-    $('.save-roles').on('click', function()
```

Comment:
if this is too controversial to remove all this I can just use the edit route instead of adding update. however I searched and could not find evidence of this button existing so I assume it was an older iteration of user profile edit